src/site/site.xml

```diff
@@ -103,7 +103,7 @@
       <item name="Scala API" href="/manual/scala-api.html"/>
 
       <item name="Configuration" href="/manual/configuration.html" collapse="true">
-        <item name="Configuration Archtecture" href="/manual/configuration.html#Architecture"/>
+        <item name="Configuration Architecture" href="/manual/configuration.html#Architecture"/>
```

Comment:
This probably means that the front facing ToC is bad. This makes it harder for users to find things (it's at least screen reader hostile, but also problematic for anyone using search).